x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable.tsx

```diff
@@ -245,16 +253,9 @@ export class Embeddable
   };
 
   async reload() {
-    const currentTime = Date.now();
-    if (this.externalSearchContext.lastReloadRequestTime !== currentTime) {
```

Comment:
I am either missing something or I think this check didn't make sense, because I'd expect `Date.now()` on line above to always return later timestamp that is saved inside `lastReloadRequestTime`.

I decided to clean this up, pls let me know if this was actually handling some edge case

Reply:
I think it was put in place to protect from multiple synchronous reload calls, but I didn't find a case where that's a problem. It's fine from my side.